pom.xml

```diff
@@ -221,7 +221,7 @@
                 <version>${maven.surefire.plugin.version}</version>
                 <configuration combine.self="override">
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
-
+                    <runOrder>failedfirst</runOrder>
```

Comment:
It is. What are the downsides? I added it so I don't have to manually specify which test to run when I'm fixing a failed test.